Model/Car.py

```diff
@@ -49,3 +49,16 @@ def move(self, x: float, y: float, _):
         self.x = x
         self.y = y
 
+    def drive(self, x: list[float], y: list[float], theta: list[float], callback: callable):
```

Comment:
This does not work in Python. Replace the check with:

if len(x) != len(y) or len(x) != len(theta)